### PR TITLE
New EX Permits review query

### PR DESCRIPTION
**Pull request by Charlie-Henry**
Adding a new query that gets the completed EX permits and the issued and in dates. Paloma wants to report the 5 most recently approved permits to Lewis once a week.

### CODE REVIEW

File: AMANDA/amanda_to_s3.py [hunk 1]
```diff
@@ -60,7 +60,7 @@
         OR(foldertype in('RW', 'EX')
             AND STATUSCODE NOT IN(70045, 50003)
             AND INDATE >= TO_DATE('10-01-2018', 'mm-dd-yyyy')
-            AND SUBCODE NOT IN(50510)
+            AND SUBCODE NOT IN(50510, 50505)
```
Comment (Charlie-Henry) [inline AMANDA/amanda_to_s3.py:63]: ignoring film permits as well now as ROW no longer manages these.

File: AMANDA/amanda_to_s3.py [hunk 2]
```diff
@@ -130,6 +130,23 @@
         AND f.WORKCODE = 50590
         AND fa.RESULTCODE = 61510
     """,
+    "ex_permits_issued": """
+    SELECT
+        CONCAT(CONCAT(f.FOLDERYEAR, '-'), f.FOLDERSEQUENCE) AS PERMIT_ID,
```
Comment (mddilley) [inline AMANDA/amanda_to_s3.py:135]: TIL that Oracle DB concat requires nesting like this for more than 2 args. 👍

**Review by mddilley — APPROVED** [review]
Looks solid to me! 🚢

File: AMANDA/amanda_to_s3.py (hunk 2 above)
Comment (Charlie-Henry) [reply AMANDA/amanda_to_s3.py:135]: maybe there is a non-nested way haha 😅 https://ramkedem.com/en/oracle-string-concatenation/#:~:text=Oracle%20String%20concatenation%20allows%20you,concatenation%20operator%20(%7C%7C).

Comment (Charlie-Henry) [reply AMANDA/amanda_to_s3.py:135]: maybe there is a non-nested way haha 😅 https://ramkedem.com/en/oracle-string-concatenation/#:~:text=Oracle%20String%20concatenation%20allows%20you,concatenation%20operator%20(%7C%7C).

Comment (mddilley) [reply AMANDA/amanda_to_s3.py:135]: Whoa - pipes! That is pretty cool

File: AMANDA/amanda_to_s3.py (hunk 1 above)
Comment (chiaberry) [reply AMANDA/amanda_to_s3.py:63]: just being nosy, i wonder who handles the film permits now...

Comment (Charlie-Henry) [reply AMANDA/amanda_to_s3.py:63]: office of special events I believe @chiaberry

**Pull request merged**